libs/ett/auth/data-access-api/src/lib/auth-http.service.spec.ts

```diff
@@ -0,0 +1,153 @@
+/**
```

Comment:
issue:

Duplicate license headers seems to have been added here for some reason.

Reply:
Thanks, that happens some times 😊